doc/manuals/python/PythonPipelines.ipynb

```diff
@@ -0,0 +1,93 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": 3
+  },
+  "orig_nbformat": 2
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "Python Pipelining\n",
+    "=================\n",
+    "\n",
+    "The following code block provides a very basic, rudimentary programatic execution of a very simple pipeline. This boiler plate example \"borrowed\" from Tom Osika's very well put together pipeline demo found at https://github.com/tao558/kwiver_demo"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import modules needed for programatic process execution\n",
+    "from kwiver.sprokit.adapters import adapter_data_set, embedded_pipeline\n",
+    "# define path to pipeline file\n",
```

Comment:
There will be when I figure out how this will be included in Kwiver/the general docs/ how sphinx/rtd handle things like that. For now leaving this comment to remind myself to fix that.